src/osm_configurator/view/toplevelframes/calculation_frame.py

```diff
@@ -1,29 +1,32 @@
 from __future__ import annotations
 
 import customtkinter
+from typing import TYPE_CHECKING
 
 from src.osm_configurator.model.project.calculation.calculation_phase_enum import CalculationPhase
+from src.osm_configurator.model.project.calculation.calculation_state_enum import CalculationState
 from src.osm_configurator.view.activatable import Activatable
 from src.osm_configurator.view.popups.alert_pop_up import AlertPopUp
 from src.osm_configurator.view.popups.yes_no_pop_up import YesNoPopUp
 from src.osm_configurator.view.states.state_manager import StateManager
 from src.osm_configurator.control.calculation_controller_interface import ICalculationController
 from src.osm_configurator.control.data_visualization_controller_interface import IDataVisualizationController
 from src.osm_configurator.view.toplevelframes.top_level_frame import TopLevelFrame
-from src.osm_configurator.view.states.view_constants import ViewConstants
 
-from typing import TYPE_CHECKING
+# Constants
+import src.osm_configurator.view.constants.button_constants as button_constants_i
+import src.osm_configurator.view.constants.frame_constants as frame_constants_i
+import src.osm_configurator.view.constants.progress_bar_constants as progress_bar_constants_i
 
 if TYPE_CHECKING:
     from src.osm_configurator.view.activatable import Activatable
     from src.osm_configurator.view.states.state_manager import StateManager
     from src.osm_configurator.control.calculation_controller_interface import ICalculationController
     from src.osm_configurator.control.data_visualization_controller_interface import IDataVisualizationController
     from src.osm_configurator.view.toplevelframes.top_level_frame import TopLevelFrame
-    from src.osm_configurator.view.states.view_constants import ViewConstants
 
 
-class CalculationFrame(TopLevelFrame, Activatable, customtkinter.CTkToplevel):
+class CalculationFrame(TopLevelFrame, Activatable):
```

Comment:
ah yes, btw we don't need activatable here, since toplevel frame "erbs" from that, thats why there was a bug before